### ISSUE TITLE
TypeError: test() got an unexpected keyword argument 'test_dataloaders'

### ISSUE THREAD

**Issue by Skinny-Joey** [kasnerz/zeroshot-d2t-pipeline#1]
Traceback (most recent call last):
  File "./decode.py", line 105, in <module>
    trainer.test(test_dataloaders=dataloader_map[args.split](), model=di.model)
TypeError: test() got an unexpected keyword argument 'test_dataloaders'

When I run decode.py, the program returns the following error. I'm not sure if it's because of the version

**Comment by kasnerz** [comment]
Yes, it sounds like a problem with your version of Pytorch Lightning. Which version are you using? And have you tried the recommended version 1.5.10 (see `requirements.txt`)? 

From what I was able to find, it seems the keyword has changed from `test_dataloaders` in [1.5.10](https://pytorch-lightning.readthedocs.io/en/1.5.10/common/test_set.html) to `dataloaders` in the [current version](https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html#testing). So as a hotfix, you may try renaming the argument.

However, PL is going through rapid development and there may be some other subtle changes which may influence the training process (I still have in fresh memory changing the default value `max_steps` from `None` to `-1` which totally broke the scheduler semantics), so it is generally better set up a new virtual environment and use the recommended version.
